server/settings/main.py

```diff
@@ -20,7 +20,8 @@ class UnitScaleSettingsModel(BaseSettingsModel):
     apply_on_opening: bool = SettingsField(
         False, title="Apply on Opening Existing Files")
```

Comment:
If enabled it will force the unit scale ratio set in the addons settings when opening any existing workfile if not aka disabled it will leave the unit scale as is (as the user has it already set to his liking).

Does it makes more sense now?